pkgs/by-name/gr/gruvbox-kvantum/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  stdenvNoCC,
+  fetchFromGitHub,
+  lib,
+  variant ? "Gruvbox-Dark-Brown",
+}:
+let
+  pname = "gruvbox-kvantum";
+in
+lib.checkListOfEnum "${pname}: variant"
+  [
+    "Gruvbox-Dark-Blue"
+    "Gruvbox-Dark-Brown"
+    "Gruvbox-Dark-Green"
+    "Gruvbox_Light_Blue"
+    "Gruvbox_Light_Brown"
+    "Gruvbox_Light_Green"
+  ]
```

Comment:
Probably it's good idea to rename light themes so it won't contain underscores, but I'm not sure